Source/Core/Core/BootManager.cpp

```diff
@@ -168,6 +170,12 @@ bool BootCore(const std::string& _rFilename)
 		VideoBackend::ActivateBackend(StartUp.m_strVideoBackend);
 		core_section->Get("GPUDeterminismMode", &StartUp.m_strGPUDeterminismMode, StartUp.m_strGPUDeterminismMode);
 
+		// Some NTSC games crash if the PAL60 option is enabled
```

Comment:
For future people reading the code, add some examples (e.g. Doc Louis's Punch-Out!!, 1942).